.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: java
 install: true
-script: mvn -Pfast-tests -Pextra-tests -B install
+script: mvn -Pfast-tests -Pextra-tests -B test
```

Comment:
I'm confused about this in regard to the previous commit. Did you mean -Pextra-tests

Reply:
The two commits are independent.  One is to change 'install' to 'test'.  The other is to remove '-Pextra-tests' as it's not actually doing anything in the build as far as I can tell.

Reply:
I told you in private.  That is to make sure the extra tests will compile.

Reply:
I pulled that commit out so -Pextra-tests will remain in place.